editor/store/test/selectors.js

```diff
@@ -88,14 +90,12 @@ describe( 'selectors', () => {
 			keywords: [ 'testing' ],
 			useOnce: true,
 		} );
+
+		cachedSelectors = filter( selectors, property( 'clear' ) );
```

Comment:
Technically you could put it out of `beforeAll` and make it `const`.

Reply:
Yeah, if I'm being honest, I'm not sure what difference is had between putting something in `beforeAll` vs. the top-level `describe` scope, so I was inclined to keep this as part of the lifecycle of the tests actually being run (assuming that's a valid distinction).

Reply:
Agreed, it's not clear. When `describe` block is processed it executes all code in the scope and schedules all those special functions for future. So this is undocumented `beforeBeforeAll` step :) I put more things in the top-level `describe` these days. One use case when you would always use `beforeAll` is when you need to finish an async task.